lib/sles4sap.pm

```diff
@@ -77,10 +78,20 @@ sub test_pids_max {
     my $rc1 = script_run "systemd-run --slice user -qt su - $sapadmin -c 'cat /sys/fs/cgroup/pids/user.slice/user-${uid}.slice/pids.max' | tr -d '\\r' | grep -qx max";
     # nproc should be set to "unlimited" in /etc/security/limits.d/99-sapsys.conf
     # Check that nproc * 2 == threads-max
-    my $rc2 = script_run "[[ \$(( \$(systemd-run --slice user -qt su - $sapadmin -c 'ulimit -u' -s /bin/bash | tr -d '\\r') * 2)) = \$(sysctl -n kernel.threads-max) ]]";
+    my $rc2 = script_run "[[ \$(( \$(systemd-run --slice user -qt su - $sapadmin -c 'ulimit -u' -s /bin/bash | tr -d '\\r') * 2 + 1)) -ge \$(sysctl -n kernel.threads-max) ]]";
     record_soft_failure "bsc#1031355" if ($rc1 or $rc2);
 }
 
+sub test_forkbomb {
+    # DO NOT call this function from a VM
```

Comment:
Why? Is it bad?

Reply:
I added some documentation in the function.  Strange, the first forkbomb can create as many as 3x times processes as the second call in the qemu backend.